rocAL/rocAL/include/augmentations/geometry_augmentations/node_rotate.h

```diff
@@ -25,14 +25,15 @@ THE SOFTWARE.
 #include "parameter_factory.h"
 #include "parameter_vx.h"
 #include "graph.h"
+#include "rocal_api_types.h"
 
 class RotateNode : public Node
 {
 public:
     RotateNode(const std::vector<Tensor *> &inputs, const std::vector<Tensor *> &outputs);
     RotateNode() = delete;
```

Comment:
Add a destructor like other nodes

Reply:
Destructor added only for crop related nodes as crop tensor is involved